app/javascript/helpers/api_routes.js

```diff
@@ -18,7 +18,8 @@ class ApiRoutes {
 
   get users() {
     return {
-      delete: () => '/users/sign_out'
+      // create: () => '/users/sign_in',
```

Comment:
Use this file instead of hard-coding the link.

Reply:
added a sessions.create() function but it breaks everything ): using literal route for now, hopefully we will figure out why putting it in api_routes doesn't work